_summaries/sukhbaatar2015end-to-end_2.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{sukhbaatar2015end-to-end}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: mmcinnestaylor # Your GitHub id
+score: 8 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+* End-To-End Memory Networks are a type of Memory Network with a possibly large external memory which is are trained end-to-end.
+* End-to-end training requires less supervision, thus improving applicability in real world settings.
+
+## Technical Implementation
+![Architecture](sukhbaatar2015end-to-end_2_b.png) 
+* The model accepts a discrete set of inputs (a story) along with a query, and outputs an answer.  
+![Q&A](sukhbaatar2015end-to-end_2_a.png)  
+* The inputs {x} and query q are transformed using embedding matrices A and B, respectively.  
```

Comment:
Formatting issue (maybe): Did you mean to put {} around the x or did you want the latex styling $$x$$?
<img width="706" alt="Screen Shot 2021-09-25 at 3 02 30 PM" src="https://user-images.githubusercontent.com/13022900/134784679-56960a78-57fa-4c51-b2e9-dc6bccee6a6c.png">

Totally fine either way, just wasn't sure if it was intended.

Reply:
It might help to briefly explain what x, q, and a are all equal to given the example image.  Like X is the "story" or "memories", q is the "input query" and a is the "answer to the query given the memories"

Reply:
@Zayne-sprague For the {} notation: the markdown/latex renderer I'm using (macos + xcode) displays the $$x$$ on a newline by itself. $x$ will render it in line on my local machine, but the renderer that the course website uses doesn't seem to work with the latter formatting. Any suggestions?

Good suggestion on clarifying the notation

Reply:
Hmm, I am using Jekyll from the readme 

`* The inputs $$x_i$$ and query q are transformed using input embedding matrices A and B, respectively. The embedded inputs are 'memories' in the embedding space.
  $$m_i = \text{Embed}_A(x_i)\qquad u = \text{Embed}_B(q)$$`

Ends up looking like
<img width="1513" alt="Screen Shot 2021-09-26 at 11 11 00 AM" src="https://user-images.githubusercontent.com/13022900/134815515-3c9494a6-db4f-4f56-b63d-e5bb4b02e7a4.png">

The interesting part being the x_i.

I think it might be differences in the rendering software.

Using $x_i$ does no formatting on my end (I see it just like its typed unfortunately)